charts/tekton-apps/templates/eventlistener.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: triggers.tekton.dev/v1beta1
 kind: EventListener
 metadata:
-  name: build-pipeline-event-listener
+  name: build-pipeline-event-listener{{ .Values.eventlistener.suffix }}
```

Comment:
make it snippet template please
see an example

```go
{{- define "tekton-apps.resourceName" -}}
{{- if .suffix }}
{{- printf "%s-%s-%s-%s" .project.project .component.name .environment .suffix }}
{{- else }}
{{- printf "%s-%s-%s" .project.project .component.name .environment }}
{{- end }}
{{- end }}
```

so use the same approach to define the listener name